package.json

```diff
@@ -86,6 +86,7 @@
     "pack-n-play": "^1.0.0-2",
     "proxyquire": "^2.0.0",
     "sinon": "^14.0.0",
+    "snap-shot-it": "^7.9.1",
```

Comment:
I would add an additional npm script that allows you to take snapshots retake snapshots, otherwise if the structure of responses changes in the future someone needs to read the docs for `snap-shot-it`, something like:

```
{"scripts: {
  "test:snap": "COMMAND TO TAKE SNAPSHOT"
}}
```

Reply:
Script added.